pkg/apis/componentconfig/v1alpha1/defaults.go

```diff
@@ -112,3 +132,193 @@ func SetDefaults_LeaderElectionConfiguration(obj *LeaderElectionConfiguration) {
 		obj.RetryPeriod = unversioned.Duration{Duration: 2 * time.Second}
 	}
 }
+
+func SetDefaults_KubeletConfiguration(obj *KubeletConfiguration) {
```

Comment:
@mikedanese We currently have scripts that auto-generate documentation based on flag defaults. We should ideally update those scripts to use the defaults from here.